sections/main-search.liquid

```diff
@@ -242,6 +242,7 @@
                                 <div class="card__badge {{ settings.badge_position }}">
                                   <span class="badge color-background-1">{{ 'templates.search.page' | t }}</span>
                                 </div>
+                              </div>
```

Comment:
Hey @jameshalldev, thanks for taking the time clean up some of our code 🙂 

While we're at it, do you mind fixing the indentation as well ? Looks like `<div class="card__content">` is indented one time too many 👍